tests/foreman/ui/test_lifecycleenvironment.py

```diff
@@ -154,3 +159,34 @@ def test_positive_create_environment_after_host_register(self):
 
         @caseautomation: notautomated
         """
+
+    @tier1
+    @skip_if_bug_open('bugzilla', 1295922)
+    def test_positive_env_list_fits_screen(self):
+        """Check if long list of lifecycle environments fits into screen
+
+        @id: 63b985b0-c847-11e6-92ad-68f72889dc7f
+
+        @Setup: save 8+ chained lifecycles environments
+
+        @BZ: 1295922
+
+        @Assert: lifecycle enviroments table fits screen
+
+        """
+        with Session(self.browser) as session:
+            env_names = [gen_alpha() for _ in range(11)]
+            for name, prior in zip(env_names, chain([None], env_names)):
+                make_lifecycle_environment(
+                    session,
+                    org=self.org_name,
+                    name=name,
+                    prior=prior
+                )
+            envs_table = session.nav.wait_until_element(
+                locator=locators['content_env.table']
+            )
+            table_width = envs_table.size['width']
+            body = session.nav.find_element(common_locators['body'])
+            body_width = body.size['width']
+            self.assertGreaterEqual(body_width - table_width, 0)
```

Comment:
The problem is that they fixed this issue changing css of table to 100% (widh and max-width). So I would need a 6.1 box to test it. Going to check if anyone has  a box like that.